### ISSUE TITLE
datamodell: vw_tww_wastewater_structure has twice the calculated fields _usage_current etc.

### ISSUE THREAD

**Issue by urskaufmann** [teksi/wastewater#66]
Table wastewater_structure has calculated fields _usage_current and _function_hierarchic.  -> they appear in vw_tww_wastewater_structure as _channel_usage_current and _channel_function_hierarchic.

Table wastewater_node has calculated fields  _status, _usage_current and _function_hierarchic  -> they appear in vw_wastewater_node as wn__status, wn__usage_current and wn__function_hierarchic. This fields are useful in vw_wastewater_node, but they are double in vw_tww_wastewater_structure!

Are there two triggers to calculate these fields twice, so it slows down? 
Is it possible/necessary to optimize?

I think, it's faster to have the calculation just to the node (channel>reach>reachpoint>node) than to the ww_structure (channel>reach>reachpoint>node>structure).

**Comment by cymed** [comment]
Good point. We could omit all calls to tww_app.update_wastewater_structure_symbology from [symbology_functions.sql]( https://github.com/teksi/wastewater/blob/main/datamodel/app/symbology_functions.sql)
